wordpress-stubs.php

```diff
@@ -85557,6 +85583,18 @@ function admin_created_user_email($text)
     function wp_is_authorize_application_password_request_valid($request, $user)
     {
     }
+    /**
+     * Validates the redirect URL protocol scheme. The protocol can be anything except http and javascript.
+     *
+     * @since 6.3.2
```

Comment:
There must be something wrong here ...

Reply:
The commit says _Generate stubs for WordPress 6.2.3_

@swissspidy Please advise, you are The WordPress Superhero.

Reply:
@swissspidy Please says something. Is this a core error?